README.md

```diff
@@ -73,12 +73,56 @@ gulp.task('default', gulp.series('clean', 'all'));
 
 ## Incremental Builds
 
-We recommend these plugins:
+You can filter out unchanged files between runs of a task using 
+`gulp.src` function's `since` option and `gulp.lastRun`:
```

Comment:
The `since` option to `gulp.src` and `gulp.lastRun`

Seems to fit better to me